config/set/php81.php

```diff
@@ -13,10 +13,8 @@
 use Rector\Php81\Rector\FunctionLike\IntersectionTypesRector;
 use Rector\Php81\Rector\MethodCall\MyCLabsMethodCallToEnumConstRector;
 use Rector\Php81\Rector\Property\ReadOnlyPropertyRector;
-use Rector\TypeDeclaration\Rector\ClassMethod\ReturnNeverTypeRector;
 
 return static function (RectorConfig $rectorConfig): void {
-    $rectorConfig->rule(ReturnNeverTypeRector::class);
```

Comment:
never type is exists on php 8.1 https://php.watch/versions/8.1/never-return-type

Reply:
it is on `TypeDeclaration`, not `CodingStyle`. It is under `TypeDeclaration` because it has behaviour:

- make never type on php 8.1
- make `@return never` on php < 8.1